src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String name;
+    private Engine engine;
+    private String color;
+    private int doorCount;
+    private int issueYear;
 
+    public Car(String name, Engine engine, String color, int doorCount, int issueYear) {
+    }
```

Comment:
you have got some values in your constructor. What next we supposed to do  with them?

Reply:
Fixed. Done